cmds/exp/disk_unlock/disk_unlock.go

```diff
@@ -171,10 +179,16 @@ func main() {
 			continue
 		}
 
-		if err := sgdisk.Unlock((string)(key), false); err != nil {
-			log.Printf("Couldn't unlock disk with HSS %d: %v", i, err)
-		} else {
-			unlocked = true
+		for r := 0; r < *retries; r++ {
+			if err := sgdisk.Unlock((string)(key), false); err != nil {
+				log.Printf("Couldn't unlock disk with HSS %d: %v", i, err)
+			} else {
+				unlocked = true
+				break
```

Comment:
To simplify breaking out of both for loops, can you put them into a separate function?

```
func TryMultipleHSSUnlocks(hssList [][]uint8{}) bool {
   ...
}
```

Then just simply `return true` as soon as one unlocks.

Reply:
so you can use a break to a label ...
https://www.ardanlabs.com/blog/2013/11/label-breaks-in-go.html

Reply:
Added a break to label!